app/views/nouns/show.html.haml

```diff
@@ -66,7 +66,9 @@
   = two_column_card t('words.show.example_sentences.title'), t('words.show.example_sentences.description') do
     = box padding: false, class: 'striped' do
       - @noun.example_sentences.each do |sentence|
-        .p-4= sentence
+        .p-4
+          = sentence
+          .align-middle.inline-block.ml-5= render 'words/tts', word: @noun, sentence: sentence
```

Comment:
What about the following? It pushes the audio to the very right.

![screengrab-20230225-1706](https://user-images.githubusercontent.com/1394828/221367207-19857a15-a2c6-4195-b50b-396df62dfee9.gif)


```suggestion
        .p-4.flex.flex-wrap.justify-between.gap-4.items-center
          = sentence
          = render 'words/tts', word: @noun, sentence: sentence

```

Reply:
Yes, that looks better :) Thanks